hydra-cluster/test/Test/Hydra/Cluster/FaucetSpec.hs

```diff
@@ -13,12 +13,11 @@ import Test.QuickCheck (generate)
 
 spec :: Spec
 spec =
-  describe "seed from faucet" $
```

Comment:
This describe was actually useful to have! It's about one specific function of the Faucet.